src/binaryen-c.h

```diff
@@ -1180,6 +1180,15 @@ BINARYEN_API void BinaryenSetFunctionTable(BinaryenModuleRef module,
                                            const char** funcNames,
                                            BinaryenIndex numFuncNames,
                                            BinaryenExpressionRef offset);
+BINARYEN_API int BinaryenIsFunctionTableImported(BinaryenModuleRef module);
+BINARYEN_API BinaryenIndex
+BinaryenGetFunctionTableNumSegments(BinaryenModuleRef module);
```

Comment:
The corresponding memory function is `BinaryenGetNumMemorySegments`, so how about calling this `BinaryenGetNumFunctionTableSegments` for consistency?